### PR TITLE
Add step to enable monitoring

### PR DESCRIPTION
**Pull request by micheleRP**
fixes https://github.com/redpanda-data/documentation-private/issues/2144

- preview Deploy for Production: [Enable monitoring](https://deploy-preview-180--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/manual/production/production-deployment/#enable-monitoring) and [Next Steps](https://deploy-preview-180--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/manual/production/production-deployment/#next-steps)

- preview Automated Deployment: [Next steps](https://deploy-preview-180--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/manual/production/production-deployment-automation/#next-steps )

### CODE REVIEW

File: modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment.adoc [hunk 1]
```diff
@@ -233,6 +233,10 @@ To create a topic:
 rpk topic create panda
 ----
 
+=== Enable monitoring
+
+xref:manage:monitoring.adoc[]. Observability is essential in production environments.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment.adoc:238]: Curious why the link comes first here, but second in the other updates?

**Review by Feediver1 — APPROVED** [review]

**Review by mattschumpert — APPROVED** [review]
Looks good for Production Deployment, but I noticed in the preview  the deploy for **development** page talks about production environments and that page is about non-prod so we should probably just say 'Enable monitoring' and link to The Monitor Redpanda page without saying production there

**Comment by micheleRP** [comment]
> Looks good for Production Deployment, but I noticed in the preview the deploy for **development** page talks about production environments and that page is about non-prod so we should probably just say 'Enable monitoring' and link to The Monitor Redpanda page without saying production there

@mattschumpert: This is the [Automated Deployment](https://deploy-preview-180--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/manual/production/production-deployment-automation/#next-steps) page (not [Deploy for Development](https://deploy-preview-180--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/manual/production/dev-deployment/)). Originally, Automated Deployment was written for dev environments, but now it's written for production. (e.g., _This recommended automated deployment provides a production-usable way to deploy and maintain a cluster._) Should I add this link to the Deploy for Development page too?

File: modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment.adoc (hunk 1 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment.adoc:238]: @Feediver1 just to make the 2 links in Next steps parallel. (The first one was already there.)

**Pull request merged**